app/src/main/java/ro/code4/monitorizarevot/db/Data.java

```diff
@@ -223,6 +240,46 @@ private String getCityBranchId(Integer questionId) {
         return branchQuestionAnswers;
     }
 
+    public List<QuestionAnswer> getUnsyncedQuestionAnswersFromForm() {
```

Comment:
please rename this method to make it more clear - suggested: getUnsyncedQuestionAnswersFromAllForms